<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DftTab</class>
 <widget class="QWidget" name="DftTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DftTab</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="displayComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>create complex signal from real</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>load complex signal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QChartView" name="upperChart"/>
     </item>
     <item>
      <widget class="QChartView" name="lowerChart"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>save complex signal</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts/QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
